kong/init.lua

```diff
@@ -802,7 +802,10 @@ function Kong.init_worker()
   runloop.init_worker.after()
 
   if is_not_control_plane then
-    plugin_servers.start()
+    if worker_id() == 0 then
```

Comment:
I think we should put `worker_id() == 0` condition in `plugin_servers/init.lua`.

Because there is no worker id check in `exit_woker`, now it is a bit asymmetry.

The code could be 
```
    plugin_servers.start()    -- check worker id in the function
    plugin_servers.check()
```

Reply:
A function named `start` but it does not start anything in some conditions...doesn't seem right.

Reply:
The original code of `start` has that logic: 
check worker id and if it is not 0 then exit.

Reply:
my previous comment is not clear, we should do this:
in kong/init.lua call `plugin_server.start`
in plugin_server/init.lua `start` function:

```lua
function plugin_servers.start()
  if worker id == 0 then
    start server
  end

  start check timer
end
```

Reply:
Yes, I think this is the right way.